Deprecate `tokenization` flag (#244)
cossacklabs/product-docs: content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md

        ##
        # Tokenization
        ##
        # [optional] [required_with=tokenized (before 0.93.0)]
        token_type: "<email|str|bytes|int32|int64>"

>         # [optinal] [deprecated since 0.93.0] [default=false] [required_with=token_type]  

Reviewer: ```suggestion
        # [optional] [deprecated since 0.93.0] [default=false] [required_with=token_type]  
```

Type `bytes` accepts string literals with base64 values. For example to set binary array `{0, 1, 2, 3}` we should 
encode it to base64: `"AAECAw=="`. AcraServer will decode base64 values and pass it as binary array according to database's protocol.

#### **tokenized** (deprecated since 0.93.0)

Required: `false`

Type: `boolean`

Group: `tokenization`

> Description: turns on tokenization for column. Deprecated since 0.93.0 - use non-empty `token_type` instead.

Reviewer: ```suggestion
Description: turns on tokenization for column. Since Acra 0.93.0 this flag is deprecated. It's enough to specify `token_type` value for a column to enable tokenization for that column with specified data type.
```

Values: `str`, `email`, `int64`, `int32`, `bytes`, `int32_string`, `int64_string`

> Depends on: `tokenized` (before 0.93.0)

Reviewer: ```suggestion
Depends on: nothing (since 0.93.0), previously `tokenized` (before 0.93.0)
```

Group: `tokenization`

> Description: configures type of [tokens](/acra/security-controls/tokenization/). Works only together with `tokenized` option (before 0.93.0). 

Reviewer: ```suggestion
Description: configures type of [tokens](/acra/security-controls/tokenization/). Configure a data type to enable tokenization for a column, Acra will create tokens with specified data type. Before Acra 0.93.0 `token_type` worked only if `tokenized` option was enabled. Since 0.93.0, `token_type` doesn't require `tokenized` flag at all to simplify the config file.
```

Type: `boolean`

> Depends on: `token_type`

Reviewer: ```suggestion
Depends on: `token_type` (since 0.93.0), previously `tokenized` (before 0.93.0).
```
